packages/gator-permissions-snap/src/permissions/erc20TokenPeriodic/types.ts

```diff
@@ -37,9 +39,24 @@ export const zErc20TokenPeriodicPermission = zPermission.extend({
     zMetaMaskPermissionData,
     z.object({
       periodAmount: zHexStr,
-      periodDuration: z.number(),
-      startTime: z.number(),
-      tokenAddress: zHexStr,
+      periodDuration: z.number().int().positive(),
+      startTime: z
+        .number()
+        .int()
+        .nullable()
```

Comment:
We could chain a `.positive()` here.

I'm not sure if `validateStartTimeZod` will catch negative numbers, but seems worthwhile to use the "native"
 zod validation as much as possible.